### DISCUSSION TITLE
Keep Christmas music and all other music in two 'libraries'

### DISCUSSION THREAD

**Discussion by johnsturgeon** [beetbox/beets#5100] [Q&A] [answered]
All my music is in Plex, and I'm interested in keeping my Christmas music in a different library.

What that means in practice is that my music is separated at the TOP level 
```
/ChristmasMusic/Andy Williams
...
/Music/Andy Williams
```
I am just getting started with beets, what I think is that it looks like I have two `directories`.. but beets wants just one?  Is that right?

**Comment by Serene-Arc** [comment]
My recommendation (and what I do myself) is that if you want a specialised library, use docker. Running a docker image of beets will keep everything separate and you don't have to worry about keeping the configurations separate.

**Comment by johnsturgeon** [comment]
ok, so that's interesting.  So another option I guess would be to swap config files?

**Reply by arsaboo** [reply, marked as answer]
Yes, you can use separate config files for different libraries. That's what I do.

**Comment by JOJ0** [comment]
Some shell aliases might help as well for switching configs

```
alias bxmas="beet -c ~/.config/bxmas/config.yaml"
```

or changing $BEETSDIR using a shell alias, that's what I do :-)

**Reply by arsaboo** [reply]
Whenever in doubt...follow what @JOJ0 does 😊
